include/dd/Package.hpp

```diff
@@ -1903,10 +1903,16 @@ template <class Config> class Package {
     const auto eliminate = std::vector<bool>(nqubits, true);
     return trace(a, eliminate).w;
   }
-  bool isCloseToIdentity(const mEdge& m, const dd::fp tol = 1e-10) {
+
+  // if checkCloseToOne is false, it only checks if it's close to a diagonal
+  // matrix
```

Comment:
Lately I started making it a best practice that whenever I touch a certain function in the project or add a new one, I also add a proper docstring to that function.
This would also be great here.